### ISSUE TITLE
Installation instructions from Getting Started not working

### ISSUE THREAD

**Issue by jonatasbaldin** [external-secrets/external-secrets#85]
**Describe the solution you'd like**
I'd like to have working installation instructions from the [Getting Started](https://external-secrets.io/guides-getting-started/) guide.

**What is the added value?**
Make a better initial developer experience.

The current command under `Install the controller` actually fails:
```
kubectl kustomize "github.com/external-secrets/external-secrets/config/default" \
| kubectl apply -f -
```

```
Failed to pull image "controller:latest": rpc error: code = Unknown desc = failed to pull and unpack image "docker.io/library/controller:latest": failed to resolve reference "docker.io/library/controller:latest": pull access denied, repository does not exist or may require authorization: server message: insufficient_scope: authorization failed
```

This is because the image name inside the `config/manager/manager.yaml` is `controller:latest`.

From the top of my head, we can fix this in some ways:
- Change the `config/manager/kustomization.yaml` file to replace the image name with the latest version
- Change directly the `config/manager/manager.yaml` to run the public `latest` image
- Change the installation instructions to set an image version, like the [etcd Operatior](https://github.com/improbable-eng/etcd-cluster-operator/blob/master/docs/installing.md#operator)

What do you think? I can work on any of the changes ✨

**Comment by jonatasbaldin** [comment]
Related to #92

**Comment by jonatasbaldin** [comment]
Fixed on #92.
